jerry-ext/extfunc/extfunc-print.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/extfunc.h"
```

Comment:
For me, extfunc stood for "external functions" (not extra). I'm happy to change, but let's look for a name that's low on spaces, hyphens, and underscores. `jerryx_common_js_funcs_` prefix for all the functions will be hard to use. (It seemed to me that the `jerryx_XXX_` prefix and the `jerryscript-ext/XXX.h` names should highly correlate.)